### ISSUE TITLE
Customize Visibility options for non-Admin depositors

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#242]
For the OpenEmory self deposit process, depositors should be allowed to select one of 2 options only:

Public
Embargo

Admin users, however, still need the ability to set a Work to Private if for some reason it needs to be withdrawn.

With the default options available in the Create/Edit Admin Set UI, we cannot enable this particular combination. (see screenshots below). If Public and Embargo are enabled for visibility and release options, an Admin user cannot set a work to Private (unless settings are enabled to allow depositors to choose any visibility level). 

To investigate:

1. Can we customize the Admin Set behaviors in code (beyond what the UI allows)?
2. Can we implement a workaround so that the other standard Visibility options are not visible to self-depositors? (this may be the easier path)
3. Is there another way we can ensure Admin users can override the Admin Set's rules so that an Admin can always set a work to Private?

Screenshot showing Visibility options we do not want non-admin users to be able to assign:

<img width="742" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelNaQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4f150494e4f79d677472575ed77572f58a650456/Visibility.png" alt="Visibility.png" />

**Comment by eporter23** [comment]
Discussed pursuing item 2 above.

**Comment by bwatson78** [comment]
PR made: https://github.com/emory-libraries/dlp-selfdeposit/pull/252

**Comment by eporter23** [comment]
@bwatson78 looks great! When I tested this as basic_user I was not presented with any options except Public and Embargo. One thing I caught during testing though is that Hyrax defaults to "Private" as the Visibility, and I clicked save too fast and the work got created as Private. If you want me to write up a new ticket to work around this let me know, but I wonder if it's possible to set the default to Public for the Publication form?
